src/IGC/IGCFix.cpp

```diff
@@ -36,7 +36,7 @@ IGCFix::Apply(const NMEAInfo &basic)
 
   /* "Use A for a 3D fix and V for a 2D fix (no GPS altitude) or for
      no GPS data" */
-  gps_valid = basic.location_available && basic.gps_altitude_available && !(basic.fix_2d);
+  gps_valid = basic.location_available && basic.gps_altitude_available && !basic.fix_2d;
```

Comment:
Why?

Reply:
I have no experience with github and haven't done any C programming in 30 years. Eventually found the compiler output , identified the error and corrected it. Sorry about all the unnecessary commits.

Reply:
Your commit contains no documentation on which error is corrected by this.
I don't think the parantheses are necessary, and they shouldn't be used; but your PR has one commit adding them, and one removing them. For no obvious reason.